pkg/operator/controller/ingress/controller_test.go

```diff
@@ -12,6 +12,10 @@ func TestTLSProfileSpecForSecurityProfile(t *testing.T) {
 	invalidTLSVersion := configv1.TLSProtocolVersion("abc")
 	invalidCiphers := []string{"ECDHE-ECDSA-AES256-GCM-SHA384", "invalid cipher"}
 	validCiphers := []string{"ECDHE-ECDSA-AES256-GCM-SHA384"}
+	tlsVersion13Ciphers := []string{"TLS_AES_128_GCM_SHA256", "TLS_AES_256_GCM_SHA384", "TLS_CHACHA20_POLY1305_SHA256",
```

Comment:
Should this be a shared constant?

Reply:
Currently `tlsVersion13Ciphers` is only being used by this test. The list of ciphers is represented as a `sets.String` in `/ingress/controller.go` to avoid using a nested `for` loop to filter any tls v1.3 ciphers from a `Custom` profile.